src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts

```diff
@@ -77,6 +79,35 @@ export class HuiStackCardEditor extends LitElement
 
     return html`
       <div class="card-config">
+        ${this._config.type === "grid"
```

Comment:
We should create a new editor for the grid card, we can still use the logic of the stack card editors, you could extend it.